ceremonies/src/tests.rs

```diff
@@ -3234,3 +3234,80 @@ fn is_endorsed_works() {
 		assert_eq!(EncointerCeremonies::is_endorsed(&alice, &(cid, 9)), None);
 	});
 }
+
+#[test]
+fn participants_assigned_mateches_participants_registered() {
```

Comment:
There is a typo.

And please link the issue that led to the introduction of this test.